e2e_test/src/coordinator_tests.rs

```diff
@@ -0,0 +1,379 @@
+        );
+        // create db
+        let _rsp = cli
+            .write::<command::TenaneMetaDataResp>(&req)
+            .await
+            .unwrap();
+        let api = "http://127.0.0.1:8901/debug".to_string();
+        let output = Command::new("curl")
+            .args([&api])
+            .output()
+            .expect("failed to execute process");
+        assert!(output.status.code() == Some(0));
+        print!("output: {}", String::from_utf8_lossy(&output.stdout));
+        assert!(String::from_utf8_lossy(&output.stdout).contains("\"time_num\":1"));
+        // write data
+        let api: String = "http://127.0.0.1:8902/api/v1/write?db=db1".to_string();
+        let output = Command::new("curl")
+            .args([
+                "-i",
+                "-u",
+                "root:",
+                "-XPOST",
+                &api,
+                "-d",
+                "tb1,tag1=tag1,tag2=tag2 field1=1 1683259054000000000",
+            ])
+            .output()
+            .expect("failed to execute process");
+        assert!(output.status.code() == Some(0));
+        assert!(
+            String::from_utf8_lossy(&output.stdout).contains("write expired time data not permit")
+        );
+
+        let output = Command::new("curl")
+            .args([
+                "-i",
+                "-u",
+                "root:",
+                "-XPOST",
+                &api,
+                "-d",
+                "tb1,tag1=tag1,tag2=tag2 field1=2",
+            ])
+            .output()
+            .expect("failed to execute process");
+        assert!(output.status.code() == Some(0));
+        print!("output: {}", String::from_utf8_lossy(&output.stdout));
+        assert!(String::from_utf8_lossy(&output.stdout).contains("200 OK"));
+        // clean env
+        kill_process("cnosdb-meta");
+        kill_process("cnosdb");
+    }
+
+    #[tokio::test]
+    async fn test_balance() {
+        // clean env
+        kill_process("cnosdb-meta");
+        kill_process("cnosdb");
+        // start cluster
+        let opt: Output = start_cluster().await;
+        assert!(opt.status.code() == Some(0));
+        // database option schema
+        let dboption = DatabaseOptions::new(
+            Duration::new("1"),
+            Some(2),
+            Duration::new("365"),
+            Some(2),
+            Some(Precision::NS),
+        );
+        let dbschema = DatabaseSchema::new_with_options("cnosdb", "db1", dboption);
+        let cli = client::MetaHttpClient::new("127.0.0.1:8901".to_string());
+
+        let req = command::WriteCommand::CreateDB(
+            "cluster_xxx".to_string(),
+            "cnosdb".to_string(),
+            dbschema,
+        );
+        // create db
+        let _rsp = cli
+            .write::<command::TenaneMetaDataResp>(&req)
+            .await
+            .unwrap();
+
+        let api = "http://127.0.0.1:8902/api/v1/write?db=db1".to_string();
+        for j in 0..10000 {
```

Comment:
It takes so much time to write 10000 points, I suggest crating 10 threads to write these points, and create a reusable HTTP client to do the requests.